planetary-social/nos
# Fix galleries expand past the width of the screen when the links are many

## Issues covered
[#24](https://github.com/verse-pbc/issues/issues/24)

## Description
This pull request fixes an issue where galleries overflow the screen when there are many links. It updates the layout of the gallery dot indicator to ensure the dots stay within note card's bounds.

## How to test
1. Go to the discover tab and search for the following notes:
     - note1r0nwluducamvwd022t9mxq5teqnxvnwqpq6zwrudgpvg7f8f795qhall8h
     - note10n297akqfrl4kkxj282hzx2l33f3ve3446l7u7gx6pmrnsh6g3zqgec5gg
2. Please check that the gallery page indicator is not over-stretching the note card.

## Screenshots/Video

Before:

<img width="442" alt="Screenshot 2024-11-15 at 5 32 33 PM" src="https://github.com/user-attachments/assets/e35e6d23-afd3-4c2d-9ab2-2fb25d98600f">

After:

https://github.com/user-attachments/assets/d25cb05f-dd2c-4b1b-ac85-716146f85fd7
